### PR TITLE
Remove Tor Desktop Shortcut

### PR DESCRIPTION
**Pull request by emtuls**
This fixes https://github.com/mandiant/VM-Packages/issues/967

### CODE REVIEW

File: packages/tor-browser.vm/tools/chocolateyinstall.ps1 [hunk 1]
```diff
@@ -12,10 +12,14 @@ try {
...
-    Remove-Item $desktopShortcut -Force -ea 0
+  # Delete Desktop shortcuts
+  $publicDesktopShortcut = Join-Path ${Env:Public} "Desktop\$toolName.lnk"
+  if (Test-Path $publicDesktopShortcut) {
+    Remove-Item $publicDesktopShortcut -Force -ea 0
+  }
+  $userDesktopShortcut = Join-Path ${Env:UserProfile} "Desktop\$toolName.lnk"
+  if (Test-Path $userDesktopShortcut) {
+    Remove-Item $userDesktopShortcut -Force -ea 0
   }
```
Comment (Ana06) [inline packages/tor-browser.vm/tools/chocolateyinstall.ps1:23]: what about avoiding code duplication using for loop. Something like:

```suggestion
  # Delete Desktop shortcuts
  ForEach ($location in @(${Env:Public}, ${Env:UserProfile}) {
    $desktopShortcut = Join-Path $location "Desktop\$toolName.lnk"
    if (Test-Path $desktopShortcut) {
      Remove-Item $desktopShortcut -Force -ea 0
    }
```

**Review by Ana06 — COMMENTED** [review]
Good catch! I wonder if this is a common enough case and we should make this a helper function.

**Comment by emtuls** [comment]
It seems useful enough IMO. I do see a few packages we have where it could be useful and it simplifies future desktop shortcut removal from guessing the right way to do it.

File: packages/common.vm/common.vm.nuspec [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>common.vm</id>
-    <version>0.0.0.20240321</version>
+    <version>0.0.0.20240402</version>
```
Comment (Ana06) [inline packages/common.vm/common.vm.nuspec:5]: You need to rebase as the version is the same as in https://github.com/mandiant/VM-Packages/pull/962/files#diff-e403a7afac0855f61eaed356192afac07ee3faf31d423f5e9afd8318ee50c188R5 recently merged

**Review by Ana06 — CHANGES_REQUESTED** [review]
Apart from the version increase, LGTM, thanks!

**Review by Ana06 — APPROVED** [review]

**Pull request merged**